scenario-api/src/main/scala/pl/touk/nussknacker/engine/graph/node.scala

```diff
@@ -298,20 +304,48 @@ object node {
   // shape of this data should probably change, currently we leave it for backward compatibility
   object FragmentInputDefinition {
 
-    @JsonCodec case class FragmentParameter(
+    @JsonCodec
+    sealed trait ValueInputWithFixedValues {
```

Comment:
this trait with a single member looks silly, but it'll be expanded with a second member (ValueInputWithFixedValuesPreset) shortly